python/cuml/cuml/manifold/t_sne.pyx

```diff
@@ -283,11 +294,18 @@ class TSNE(Base,
     learning_rate : float (default 200.0)
         The learning rate usually between (10, 1000). If this is too high,
         t-SNE could look like a cloud / ball of points.
-    n_iter : int (default 1000)
+    max_iter : int (default 1000)
         The more epochs, the more stable/accurate the final embedding.
     n_iter_without_progress : int (default 300)
         Currently unused. When the KL Divergence becomes too small after some
         iterations, terminate t-SNE early.
+    n_iter : int (default "deprecated")
```

Comment:
```suggestion
    n_iter : int (default 1000)
```
In scikit-learn we keep the docstring stating the default value that it used to be, not repeating what the actual "in Python code" default value is.